src/sentry/backup/imports.py

```diff
@@ -441,8 +441,33 @@ def import_in_global_scope(
     Sentry instance, some behaviors in this scope are different from the others. In particular,
     superuser privileges are not sanitized. This method can be thought of as a "pure"
     backup/restore, simply serializing and deserializing a (partial) snapshot of the database state.
+    During the setup of a fresh Sentry instance, there are a couple of models that are automatically created.
+    The Sentry org, a Sentry team, and an internal project. During a global import, we want to avoid persisting
+    these default models and start from scratch.
     """
 
+    from django.core.management.color import no_style
+    from django.db import connection
+
+    from sentry.models.organization import Organization
+    from sentry.models.organizationmember import OrganizationMember
+    from sentry.models.project import Project
+    from sentry.models.projectkey import ProjectKey
+    from sentry.models.team import Team
+
+    models_to_delete = [Project, ProjectKey, Organization, OrganizationMember, Team]
+
+    # Global imports will never be run in production, ever
```

Comment:
Imo this comment should say "Global imports will only ever be executed by self-hosted" instead. It's not so much whether its prod/staging/local test, as whether its self-hosted or not.